firrtl/src/main/scala/firrtl/ir/Serializer.scala

```diff
@@ -110,6 +110,8 @@ object Serializer {
       b ++= "SInt"; s(width); b ++= "(0h"; b ++= value.toString(16); b ++= ")"
     case IntegerPropertyLiteral(value) =>
       b ++= "Integer("; b ++= value.toString(10); b ++= ")"
+    case StringPropertyLiteral(value) =>
+      b ++= "String(\""; b ++= value; b ++= "\")"
```

Comment:
I had thought there would be a cleaner way of expressing this. However, the alternative actually looks worse:

```suggestion
      b ++= """String(""""; b ++= value; b ++= """")"""
```

Let's keep this as-is. 😉